migrations/v4.js

```diff
@@ -0,0 +1,123 @@
+import { describe, whereContent, whereFromPlugin, mutateContent, checkContent, updatePlugin } from 'adapt-migrations';
+
+describe('adapt-contrib-assessment - v3.0.0 > v4.3.0', async () => {
+  let articles, assessments;
+
+  whereFromPlugin('adapt-contrib-assessment - from v3.0.0', { name: 'adapt-contrib-assessment', version: '<4.3.0' });
+
+  whereContent('adapt-contrib-assessment - where assessment', async content => {
+    articles = content.filter(({ _type }) => _type === 'article');
+    assessments = articles.filter(({ _assessment }) => _assessment !== undefined);
+    return assessments.length;
+  });
+
+  /**
+   * * Add JSON field and set attribute.
+   */
+  mutateContent('adapt-contrib-assessment - add assessment._scrollToOnReset', async () => {
+    assessments.forEach(assessment => {
+      assessment._scrollToOnReset = false;
+    });
+    return true;
+  });
+
+  checkContent('adapt-contrib-assessment - check assessment._scrollToOnReset attribute', async () => {
+    const isValid = assessments.every(assessment =>
+      assessment._scrollToOnReset === false
+    );
+    if (!isValid) throw new Error('adapt-contrib-assessment - _scrollToOnReset not added to every instance of assessment');
+    return true;
+  });
+
+  updatePlugin('adapt-contrib-assessment - update to v4.3.0', { name: 'adapt-contrib-assessment', version: '4.3.0', framework: '>=5.4.0' });
+});
+
+describe('adapt-contrib-assessment - v4.3.0 > v4.4.0', async () => {
+  let course, assessmentConfig, assessments;
```

Comment:
`assessments` isn't defined in this block

Reply:
Nice one.